src/BottomTab.js

```diff
@@ -0,0 +1,77 @@
+import './BottomTab.css'
+import {AppModes, TasksShowing} from "./App";
+import {useState} from "react";
+
+function BottomTab(props) {
+    const [idsToBeDeleted, setIDsToBeDeleted] = useState(null);
+
+    return <div id = "footer">
+            {props.appMode === AppModes.EDIT_MODE ?
```

Comment:
This is such a big ternary operator it's hard to read the parts of it.

I suggest you change from:
return <div>
   {aaaa ? 
    bbbb : 
    ccc}
  </div>
to:
const deleteContainer = bbbb;
const switcherContainer = ccc;
return <div>
   {aaaa ? deleteContainer : switcherContainer}
  </div>